lib/importer/etd.rb

```diff
@@ -12,7 +12,7 @@ module Importer::ETD
   # @param [Logger] logger
   # @return [Integer]
   def self.import(meta:, data:, options: {}, logger: Logger.new(STDOUT))
-    raise ArgumentError, "Nothing found in #{data}" if data.empty?
```

Comment:
What's motivating the changes in this file? 

Should this be a separate PR? Are unit tests needed?